testdata/cases/jsonschema-number/test.schema.json

```diff
@@ -71,8 +71,8 @@
     },
     ".tests.Values.int326": {
       "type": "integer",
-      "maximum": -1,
-      "minimum": -100,
+      "maximum": -100,
+      "minimum": -1,
```

Comment:
This was my mistake. I simply copied and pasted incorrectly.

It would be good to modify the proto file like this:

```protobuf
// Inclusive Min/Max with negative value
int32 int326 = 6 [(pubg.jsonschema.field) = {numeric: {inclusive_minimum: -100, inclusive_maximum: -1}}];

// Exclusive Min/Max with negative value
int32 int327 = 7 [(pubg.jsonschema.field) = {numeric: {exclusive_minimum: -100, exclusive_maximum: -1}}];
```